packages/knit/src/schema.test.ts

```diff
@@ -672,6 +672,48 @@ describe("client", () => {
         },
       });
     });
+    test("oneof in fetch", async () => {
+      const response = await client.fetch({
+        "spec.AllService": {
+          getAll: {
+            $: {},
+            oneof: {
+              oneofValue: oneof({
+                message: {
+                  id: {},
+                },
+                nestedMessage: {
+                  nested: {
+                    // id: {},
+                  },
+                },
+              }),
+            },
+          },
+        },
+      });
+      type Actual = typeof response;
+      type Expected = {
+        "spec.AllService": {
+          getAll: {
+            oneof: {
+              oneofValue: Oneof<{
+                message: {
+                  id: string;
+                };
+                nestedMessage: {
+                  nested: {
+                    // id: string;
+                  };
+                };
+              }>;
+            };
+          };
+        };
+      };
+      type Diff = DeepDiff<Actual, Expected>;
+      expectType<Equal<Diff, never>>(true);
```

Comment:
This seems to fail. I believe the issue has to do with the `SubSet` type having problems with the oneof symbol but will need to look into it.